overthebox/files/bin/otb-subscribe

```diff
@@ -8,14 +8,18 @@ set -e
 
 otb_info "Subscribing to the overthebox service..."
 
-ubus call network.interface dump \
-	| jq '.interface // []
+ubus call network.interface dump | jq \
+	--arg board "$(cat /tmp/sysinfo/board_name)" \
+	--arg macaddr "$(cat /sys/class/net/eth0/address)" \
```

Comment:
What about ? 
root@OverTheBox:~# ls -l /sys/class/net/ | grep -v virtual | awk -F "/" '{print $NF}'
eth0

Reply:
eth0 is ok for all devices we (plan to) sell.
There is no way to know now what will be needed in the future,
so i prefer to keep it simple & stupid.